helpers/setup-spark-k8s.py

```diff
@@ -104,6 +105,17 @@ def set_up_user(username: str, name_space: str, defaults: Dict) -> None:
     os.system(f"{KUBECTL_CMD} label serviceaccount --kubeconfig={kubeconfig} --context={context_name} {username} {label} --namespace={namespace}")
     os.system(f"{KUBECTL_CMD} label rolebinding --kubeconfig={kubeconfig} --context={context_name} {rolebindingname} {label} --namespace={namespace}")
 
+def setup_spark_conf_defaults(username: str, namespace: str) -> None:
+    DYNAMIC_DEFAULTS_CONF_FILE = utils.get_dynamic_defaults_conf_file()
```

Comment:
I would also use an env variable to specify this, as we discussed in the slides. This should allow us to specify different environment, between prod, test, svil environments (that might have different resource demands...e.g. executors)